### PR TITLE
More cuisine options

### PR DESCRIPTION
**Pull request by karenav**

### CODE REVIEW

File: project/src/main/resources/com/google/sps/data/cuisinesSearchWords.json [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-    "italian": ["italian", "pasta"],
+    "italian": ["italian", "italy", "pasta"],
     "asian": ["asian", "noodles", "thai"],
     "hamburger": ["burger", "hamburger"],
     "salad": ["salad"],
     "homemade": ["homemade", "home"],
     "hummus": ["hummus"],
-    "pizza": ["pizza"]
+    "pizza": ["pizza"],
+    "mexican": ["mexican, taco, tacos, nachos"],
```
Comment (tbarnahor) [inline project/src/main/resources/com/google/sps/data/cuisinesSearchWords.json]: Make the search words seperate strings

**Review by tbarnahor — APPROVED** [review]
I approve, but just fix the comment before merging :)

File: project/src/main/resources/com/google/sps/data/cuisinesSearchWords.json (hunk 1 above)
Comment (karenav) [reply project/src/main/resources/com/google/sps/data/cuisinesSearchWords.json]: Good catch! thanks!

**Review by szwbar — APPROVED** [review]
LGTM!

**Pull request merged**